test/collection_serializer_test.rb

```diff
@@ -1,5 +1,15 @@
 require 'test_helper'
 
+class SerializedResource < ActiveModelSerializers::Model
+  # See test line 94
+end
+
+class SerializedResourceSerializer < ActiveModel::Serializer
+  def json_key
+    'resource'
+  end
+end
```

Comment:
Seems you are wanting these in the global scope. Can we perhaps dynamically create and remove these classes or alias exiting ones? (.e.g SerializedResource = Post...)